build.gradle

```diff
@@ -9,13 +9,18 @@ buildscript {
 
         // Koin
         koinVersion = '2.0.1'
+        mockkVersion = '1.9.3'
 
         // Testing
         junitVersion = "4.12"
         mockitoVersion = "2.24.0"
 
         // Extensions
-        ktxVersion = "1.1.0"
+        ktxVersion = "1.0.2"
```

Comment:
why do you downgrade it??

Reply:
It was during the rebase with master, fixing it